docs/en/operation/Performance-Tuning.md

```diff
@@ -195,7 +192,8 @@ There are 3 options for loading a missing path: `Never`, `Once`, `Always`.
 `ONCE` will use the default behavior of only scanning each directory once ever, and `NEVER` will never consult the UFS
 and thus prevent Alluxio from scanning for new files at all.
 
-The Alluxio master maintains a cache to approximate which UFS paths have been previously loaded, to approximate the `Once` behavior.
+The Alluxio master maintains a cache to approximate which UFS paths have been previously loaded,
+to approximate the `Once` behavior.
```

Comment:
`to approximate` twice in the same sentence sounds a little odd here. Can we try re-wording?

Reply:
updated